lib/measures/default_feature_reports/measure.rb

```diff
@@ -203,6 +242,682 @@ def run(runner, user_arguments)
+    west_window_area = sql_query(runner, sql_file, "InputVerificationandResultsSummary", "TableName='Window-Wall Ratio' AND RowName='Window Opening Area' AND ColumnName='West (225 to 315 deg)'").to_f
+    feature_report.program.window_area[:west_window_area] = convert_units(west_window_area, 'm^2', 'ft^2')
+    #total_window_area
+    total_window_area = north_window_area + south_window_area + east_window_area + west_window_area
+    feature_report.program.window_area[:total_window_area] = convert_units(total_window_area, 'm^2', 'ft^2')
+
+    ## wall_area
+    #north_wall_area
+    north_wall_area = sql_query(runner, sql_file, "InputVerificationandResultsSummary", "TableName='Window-Wall Ratio' AND RowName='Gross Wall Area' AND ColumnName='North (315 to 45 deg)'").to_f
+    feature_report.program.wall_area[:north_wall_area] = convert_units(north_wall_area, 'm^2', 'ft^2')
+    #south_wall_area
+    south_wall_area = sql_query(runner, sql_file, "InputVerificationandResultsSummary", "TableName='Window-Wall Ratio' AND RowName='Gross Wall Area' AND ColumnName='South (135 to 225 deg)'").to_f
+    feature_report.program.wall_area[:south_wall_area] = convert_units(south_wall_area, 'm^2', 'ft^2')
+    #east_wall_area
+    east_wall_area = sql_query(runner, sql_file, "InputVerificationandResultsSummary", "TableName='Window-Wall Ratio' AND RowName='Gross Wall Area' AND ColumnName='East (45 to 135 deg)'").to_f
+    feature_report.program.wall_area[:east_wall_area] = convert_units(east_wall_area, 'm^2', 'ft^2')
+    #west_wall_area
+    west_wall_area = sql_query(runner, sql_file, "InputVerificationandResultsSummary", "TableName='Window-Wall Ratio' AND RowName='Gross Wall Area' AND ColumnName='West (225 to 315 deg)'").to_f
+    feature_report.program.wall_area[:west_wall_area] = convert_units(west_wall_area, 'm^2', 'ft^2')
+    #total_wall_area
+    total_wall_area = north_wall_area + south_wall_area + east_wall_area + west_wall_area
+    feature_report.program.wall_area[:total_wall_area] = convert_units(total_wall_area, 'm^2', 'ft^2')
+
+
+    ##roof_area
+    #equipment_roof_area
+    ######to be added
+    #photovoltaic_roof_area
+
+    #available_roof_area
+    #available_roof_area = total_roof_area - equipment_roof_area
+
+    #total_roof_area
+    total_roof_area = 0.0
+    surfaces.each do |surface|
+      if surface.outsideBoundaryCondition == "Outdoors" and surface.surfaceType == "RoofCeiling"
+        total_roof_area += surface.netArea
+      end
+    end
+    ####another way####
+    #total_roof_area = sql_query(runner, sql_file, "InputVerificationandResultsSummary", "TableName='Skylight-Roof Ratio' AND RowName='Gross Roof Area' AND ColumnName='Total'").to_f
+    feature_report.program.roof_area[:total_roof_area] = convert_units(total_roof_area, 'm^2', 'ft^2')
+    puts total_roof_area
+
+    #orientation
+    #to check!   
+    building_rotation = model.getBuilding.northAxis
+    feature_report.program.orientation = building_rotation
```

Comment:
yeah this one is hard to define.  we really want major axis, talk to @davidgoldwasser